packages/arb-token-bridge-ui/src/util/AddressUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { Provider } from '@ethersproject/providers'
+
+export async function addressIsSmartContract(
+  address: string,
+  provider: Provider
+) {
+  try {
+    return !!((await provider?.getCode(address)).length > 2)
```

Comment:
Two things:
1) Why do we need the optional chaining at `provider?` when the passed in provider can't be nullish?
2) Seems like this should be enough: `(await provider.getCode(address)).length > 2` and we don't need the `!!` (it's probably related to the comment above)

Reply:
ah yes, that was an issue with `JsonRpcProvider` but since we've changed to `Provider`